<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>seeUser</class>
 <widget class="QMainWindow" name="seeUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: none;
color:black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>501</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Geography</string>
     </attribute>
     <widget class="QLabel" name="flagglename">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;BM Jua&quot;;</string>
      </property>
      <property name="text">
       <string>Flaggle</string>
      </property>
     </widget>
     <widget class="QLabel" name="mappername">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>401</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;BM Jua&quot;;</string>
      </property>
      <property name="text">
       <string>Mapper</string>
      </property>
     </widget>
     <widget class="QLabel" name="flaghscore">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Highest score:</string>
      </property>
     </widget>
     <widget class="QLabel" name="flagavgscore">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Average score:</string>
      </property>
     </widget>
     <widget class="QLabel" name="flagbtime">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Best time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="flagavgtime">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Average time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="maphscore">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Highest score:</string>
      </property>
     </widget>
     <widget class="QLabel" name="mapavgscore">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Average score:</string>
      </property>
     </widget>
     <widget class="QLabel" name="mapbtime">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>190</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Best time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="mapavgtime">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Average time:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Mathematics</string>
     </attribute>
     <widget class="QLabel" name="timebtime">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Best time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="timehscore">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Highest score:</string>
      </property>
     </widget>
     <widget class="QLabel" name="quickhscore">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Highest score:</string>
      </property>
     </widget>
     <widget class="QLabel" name="quickavgtime">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Average time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="quickavgscore">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Average score:</string>
      </property>
     </widget>
     <widget class="QLabel" name="timetellername">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;BM Jua&quot;;</string>
      </property>
      <property name="text">
       <string>Time Teller</string>
      </property>
     </widget>
     <widget class="QLabel" name="quickbtime">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>190</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Best time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="timeavgscore">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Average score:</string>
      </property>
     </widget>
     <widget class="QLabel" name="timeavgtime">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Average time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="quickmathname">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>401</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;BM Jua&quot;;</string>
      </property>
      <property name="text">
       <string>Quick Math</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>English</string>
     </attribute>
     <widget class="QLabel" name="spellhscore">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Highest score:</string>
      </property>
     </widget>
     <widget class="QLabel" name="hangbtime">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Best time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="spellavgtime">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Average time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="hangavgscore">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Average score:</string>
      </property>
     </widget>
     <widget class="QLabel" name="hangmanname">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;BM Jua&quot;;</string>
      </property>
      <property name="text">
       <string>Hangman</string>
      </property>
     </widget>
     <widget class="QLabel" name="spellbtime">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>190</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Best time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="hangavgtime">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Average time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="spellername">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>401</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;BM Jua&quot;;</string>
      </property>
      <property name="text">
       <string>Speller</string>
      </property>
     </widget>
     <widget class="QLabel" name="spellavgscore">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Average score:</string>
      </property>
     </widget>
     <widget class="QLabel" name="hanghscore">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Highest score:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="userName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;BM Jua&quot;;</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ACMchampion">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>82</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
image: url(:/image for c++ project/Background/trophy(black).png);
}

QLabel:hover {
image: url(:/image for c++ project/Background/trophy(black-hover).png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bestgameLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;BM Jua&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bestscoreLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;BM Jua&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="besttimeLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;BM Jua&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ACMspeedrun">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>82</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
image: url(:/image for c++ project/Background/time(black).png);
}

QLabel:hover {
image: url(:/image for c++ project/Background/time(black-hover).png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ACMturtle">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>450</y>
      <width>82</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
image: url(:/image for c++ project/Background/turtle(black).png);
}

QLabel:hover {
image: url(:/image for c++ project/Background/turtle(black-hover).png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ACMdontgiveup">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>82</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
image: url(:/image for c++ project/Background/dontgiveup(black).png);
}

QLabel:hover {
image: url(:/image for c++ project/Background/dontgiveup(black-hover).png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icons.qrc">:/image for c++ project/Background/userPage.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 30pt &quot;BM Jua&quot;;
color: black;
background: none;
border: none;
}

QPushButton:hover {
color: red;
}
</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="profilePic">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>109</y>
      <width>140</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="profilePicComboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>tabWidget</zorder>
   <zorder>userName</zorder>
   <zorder>ACMchampion</zorder>
   <zorder>bestgameLabel</zorder>
   <zorder>bestscoreLabel</zorder>
   <zorder>besttimeLabel</zorder>
   <zorder>ACMspeedrun</zorder>
   <zorder>ACMturtle</zorder>
   <zorder>ACMdontgiveup</zorder>
   <zorder>exitButton</zorder>
   <zorder>profilePic</zorder>
   <zorder>profilePicComboBox</zorder>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
